src/main/java/org/terasology/customOreGen/PocketStructureDefinition.java

```diff
@@ -113,18 +115,19 @@ public DiffusePocketStructure(Transform transform, Random random, Vector3ic chun
             if (rMax < 0) {
                 rMax = 0;
             }
-            float[] bb = new float[]{-rMax, -rMax, -rMax, rMax, rMax, rMax};
-            transform.transformBB(bb);
+            Vector3f min = new Vector3f();
+            Vector3f max = new Vector3f();
+            transform.frustumAabb(min, max);
 
-            float minX = Math.min(bb[0], bb[3]);
-            float minY = Math.min(bb[1], bb[4]);
-            float minZ = Math.min(bb[2], bb[5]);
+            float minX = Math.min(min.x, max.x);
+            float minY = Math.min(min.y, max.y);
+            float minZ = Math.min(min.z, max.z);
```

Comment:
The `Math.min`s (and `Math.max`s later) are unnecessary. The coordinates of `min` are already <= the coordinates of `max`.